web/impact/impact/v1/events/organization_created_event.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from impact.models import (
+    Organization,
+    Partner,
+    Startup,
+)
+from impact.utils import DAWN_OF_TIME
+
+
+class OrganizationCreatedEvent(object):
+    EVENT_TYPE = "created"
```

Comment:
After thinking this through, I think it's better to make the event types distinct since creating a user is pretty different from creating an organization if we ever want to combine these histories it will be valuable to have them marked distinctly.